### PR TITLE
Liberty Bot

### LINKED ISSUE #65260: Turning off TALON UGV Liberty Edition produces regular TALON UGV
### Describe the bug

Using HM12 to turn off TALON UGV Liberty Edition, produces regular TALON UGV that's with M16A4 rifle.

### Attach save file

[test-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/11302913/test-trimmed.tar.gz)

### Steps to reproduce

1. Load the save.
2. Turn off the TALON UGV Liberty Edition.
3. Find that it turned into regular TALON UGV.

### Expected behavior

There should be a TALON UGV liberty edition in item form.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.2846 (22H2)
- Game Version: 0.G-1503-ge8a1eafffb [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TealcOneill**
#### Summary
Bugfixes "Gives an explanation for why the TALON UGV Liberty Edition turns into the regular TALON UGV M16A4."

#### Purpose of change
Closes #65260

#### Describe the solution
Adds a new item and new monster that gives a slight difference from the original M16A4 Talon bot.

#### Describe alternatives you've considered
Add fireworks to the game so you can reload the Liberty Bot and allow you to activate it as the Liberty Bot instead of the regular Talon bot.

#### Testing
I loaded the game, gave myself 10 in all skills, spawned a base liberty bot, used the HM12 Dazzle rifle to disable it, deactivated it, wielded it, then reactivated it. This gave me the refurbished Liberty bot as a friendly bot.

#### Additional context
First legit PR in a long time. Woot woot

### CODE REVIEW

File: data/json/items/corpses/inactive_bots.json [hunk 1]
```diff
@@ -320,6 +320,31 @@
     "flags": [ "SINGLE_USE" ],
     "melee_damage": { "bash": 8 }
   },
+  {
+    "id": "bot_secubot_liberty",
+    "type": "TOOL",
+    "name": { "str": "inactive TALON UGV Liberty Edition" },
```
Comment (Qrox) [inline data/json/items/corpses/inactive_bots.json:326]:
```suggestion
    "name": { "str": "inactive TALON UGV Liberty Edition", "str_pl": "inactive TALON UGVs Liberty Edition" },
```

File: data/json/items/corpses/inactive_bots.json [hunk 2]
```diff
@@ -320,6 +320,31 @@
...
+    "description": "This bot has had tubes welded above the original guns that allow it to shoot fireworks in order to blind and stun attackers. With fireworks being harder to come by then the ammo for the original weapons, the best option would be to convert it back to it's original guns. It should be easy enough to give it back the use of the original M16A4.",
```
Comment (Qrox) [inline data/json/items/corpses/inactive_bots.json:327]:
```suggestion
    "description": "This bot has had tubes welded above the original guns that allow it to shoot fireworks in order to blind and stun attackers.  With fireworks being harder to come by then the ammo for the original weapons, the best option would be to convert it back to it's original guns.  It should be easy enough to give it back the use of the original M16A4.",
```

File: data/json/monsters/defense_bot.json [hunk 3]
```diff
@@ -153,6 +153,55 @@
     "flags": [ "SEES", "HEARS", "ELECTRONIC", "COLDPROOF", "NO_BREATHE", "PATH_AVOID_DANGER_1", "LOUDMOVES", "STUN_IMMUNE" ],
     "armor": { "bash": 14, "cut": 14, "bullet": 11 }
   },
+  {
+    "id": "mon_secubot_america_refurbished",
+    "type": "MONSTER",
+    "name": { "str": "refurbished autonomous TALON UGV Liberty Edition" },
```
Comment (Qrox) [inline data/json/monsters/defense_bot.json:159]:
```suggestion
    "name": { "str": "refurbished autonomous TALON UGV Liberty Edition", "str_pl": "refurbished autonomous TALON UGVs Liberty Edition" },
```

File: data/json/monsters/defense_bot.json [hunk 4]
```diff
@@ -153,6 +153,55 @@
...
+    "description": "Once a Talon UGV bot converted to shoot fireworks at trespassers, this has been refurbished and uses it's M16A4 once more. It still has the American flag decal from it's service as a trespass deterrent.",
```
Comment (Qrox) [inline data/json/monsters/defense_bot.json:160]:
```suggestion
    "description": "Once a Talon UGV bot converted to shoot fireworks at trespassers, this has been refurbished and uses it's M16A4 once more.  It still has the American flag decal from it's service as a trespass deterrent.",
```

**Comment by TealcOneill** [comment]
@Qrox Thank you for the spell checking! I'll try and remember the "str_pl" and double spaces in the future!

**Pull request merged**